video/qt_decoder.h

```diff
@@ -206,6 +212,10 @@ class QuickTimeDecoder : public VideoDecoder, public Audio::QuickTimeAudioDecode
 		kUpdateModeOffscreenOnly,
 		kUpdateModeFromOffscreen,
 		kUpdateModeDirectToScreen,
+
+		kScaleFactorOne = 1,
```

Comment:
I do not like this enum usage. It is not consistent. You are passing it as an integer but then work with the actual values. I would suggest dropping it altogether and using numbers.